README.md

```diff
@@ -149,33 +149,6 @@ Currently Java 17 and Java 11 are bundled (JDK_VERSION= {17, 11}).
 
 Full list of options be found [here](./Makefile).
 
-### Running PCT with different version of dependencies
```

Comment:
did you mean `overridenPlugins` not `includePlugins`?

`includePlugins` is still in the source https://github.com/jenkinsci/plugin-compat-tester/pull/419/files#diff-974ab2995b72e238f0b5447579e1ecd69b65f87c3970900501378daac90323ccR72-R75 and is used to split testing of plugins in the war so that testing can be paralleized

Reply:
> did you mean overridenPlugins not includePlugins?

Yes but the error was in my self-review, not in the code, so there is no code change to be made